script.js

```diff
@@ -0,0 +1,110 @@
+class BaseClass {
+    constructor(value) {
+        this.value = value;
+    }
+
+    plus(...n) {
+        for (let nums of n) this.value += nums;

```

Comment:
We have quite a lot of methods to work with arrays. I don't think you need to use it there

Reply:
It seems 'reduce' method will fit your needs here